envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -54,6 +69,27 @@ private fun getCommunicationMode(proto: Value?): CommunicationMode {
     }
 }
 
+fun Value?.toStatusCodeFilter(properties: AccessLogFilterProperties): AccessLogFilterSettings.StatusCodeFilter? {
+    val value = if (this?.stringValue != null) this.stringValue.toLowerCase() else return null
+
+    if (!properties.statusCodeFilterPattern.matches(value)) {
```

Comment:
I am not sure if `statusCodeFilterPattern` property is a good idea. If we change property I expect it to work but few lines below we have hardcoded values

Reply:
will remove those hardcoded values